integration/unplugin/src/index.ts

```diff
@@ -13,49 +13,31 @@ import {
 } from '@danielx/civet/ts-diagnostic';
 import * as fs from 'fs';
 import path from 'path';
-import ts from 'typescript';
+import type { FormatDiagnosticsHost, Diagnostic, System } from 'typescript';
```

Comment:
Should we remove `typescript` as a dependency then? Or do we use it elsewhere?

Reply:
We can remove it as a dependency, but we should add esbuild & typescript as dev dependencies for types locally.